### PR TITLE
Update native tests doc

### PR DESCRIPTION
**Pull request by alexandrasp**

### CODE REVIEW

**Review by gctucker — COMMENTED** [review]
Please add git commit messages.

**Comment by alexandrasp** [comment]
tested ok on staging: https://static.staging.kernelci.org/docs/tests/kselftest/ and https://static.staging.kernelci.org/docs/tests/ltp/

File: kernelci.org/content/en/docs/tests/ltp.md [hunk 1]
```diff
@@ -32,18 +32,23 @@ architecture.  All these tests are typically run on every kernel revision built
...
+|      bcm2836-rpi-2-b      |   arm   |    ✔   |                 |     |     |    |     |        |             |
+|      beaglebone-black     |   arm   |        |                 |     |  ✔  |    |     |        |             |
+|         hip07-d05         |  arm64  |        |                 |     |     |    |     |        |             |
+|     hp-11A-G6-EE-grunt    | x86\_64 |    ✔   |        ✔        |     |  ✔  |  ✔ |  ✔  |    ✔   |             |
+| hp-x360-12b-n4000-octopus | x86\_64 |    ✔   |                 |     |  ✔  |  ✔ |     |        |             |
+|     hp-x360-14-G1-sona    | x86\_64 |        |                 |     |     |    |     |        |             |
+|    meson-g12b-odroid-n2   |   arm   |        |                 |     |     |    |     |        |             |
+|      mt8173-elm-hana      |  arm64  |    ✔   |        ✔        |     |  ✔  |  ✔ |  ✔  |    ✔   |             |
+|        qcom-qdf2400       |   arm   |    ✔   |        ✔        |  ✔  |  ✔  |  ✔ |  ✔  |    ✔   |             |
+|       qemu\_x86\_64       | x86\_64 |        |                 |     |     |    |     |        |      ✔      |
```
Comment (gctucker) [inline kernelci.org/content/en/docs/tests/ltp.md:50]: `timers_qemu` is actually the same as `timers`, just a different LAVA template to make it run on QEMU. It's the same LTP tests, so I think the column should be grouped with `timers`.

Comment (alexandrasp) [reply kernelci.org/content/en/docs/tests/ltp.md]: updated.

**Review by gctucker — APPROVED** [review]

**Pull request merged**